### PR TITLE
SAI PTF 6: sainexthop

### PR DESCRIPTION
**Pull request by pkochmanska**

### CODE REVIEW

File: ptf/sainexthop.py [hunk 1]
```diff
@@ -0,0 +1,863 @@
...
+    '''
+        Basic L3 nexthop tests.
+    '''
+    def runTest(self):
+        self.removeNexthopTest()
+        self.cpuNexthopTest()
+
+    def removeNexthopTest(self):
+        '''
+            Test verifies correct nexthop removal.
```
Comment (ravi861) [inline ptf/sainexthop.py:38]: Can you please add some more details as to what the outcome is when nexthop is removed?

File: ptf/sainexthop.py [hunk 2]
```diff
@@ -0,0 +1,863 @@
...
+                post_stats["SAI_QUEUE_STAT_PACKETS"] -
+                pre_stats["SAI_QUEUE_STAT_PACKETS"],
+                1)
+
+        finally:
+            sai_thrift_remove_route_entry(self.client, route3)
+            sai_thrift_remove_route_entry(self.client, route2)
+            sai_thrift_remove_route_entry(self.client, route1)
+            sai_thrift_remove_neighbor_entry(self.client, neighbor_entry)
+            sai_thrift_remove_next_hop(self.client, nhop)
```
Comment (ravi861) [inline ptf/sainexthop.py:97]: Are we deleting twice?

**Review by ravi861 — APPROVED** [review]
Good to go after addressing comments

File: ptf/sainexthop.py (hunk 1 above)
Comment (rlhui) [reply ptf/sainexthop.py:39]: @pkochmanska  - please address this comment.  thanks.

**Review by rlhui — APPROVED** [review]

**Pull request merged**